libyul/backends/evm/EVMDialect.cpp

```diff
@@ -112,6 +112,27 @@ pair<YulString, BuiltinFunctionForEVM> createFunction(
 	return {name, f};
 }
 
+set<YulString> createReservedIdentifiers(bool _objectAccess)
+{
+	set<YulString> reserved;
+	for (auto const& instr: evmasm::c_instructions)
+	{
+		string name = instr.first;
+		transform(name.begin(), name.end(), name.begin(), [](unsigned char _c) { return tolower(_c); });
+		reserved.emplace(name);
+	}
+	if (_objectAccess)
```

Comment:
We could unconditionally include these making these reserved identifiers even if object access is not supported.

Reply:
I think we should reserve these unconditionally, @chriseth what do you think?

Reply:
I would include them unconditionally, even if object access is not supported.

Reply:
If that is possible in a non-breaking way....

Reply:
Let's do this.